lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/Assertions.java

```diff
@@ -0,0 +1,16 @@
+package com.devinci.lib.robolectric;
+
+import org.robolectric.shadows.ShadowLog;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Assertions {
+
+  public static void assertLogged(int type, String tag, String msg, Throwable throwable) {
```

Comment:
We can extract this later to a separate assertj module. It's usage would look something like this:
```
LogItem logItem = ShadowLog.getLogs().get(0);
assertThat(logItem).hasType(type).hasMessage(msg).hasThrowable(throwable);
```

But it's ok for now.